### ISSUE TITLE
Solargraph Documentation HoverOver Documentation Links Crash Server When Clicked On

### ISSUE THREAD

**Issue by gwillcox-r7** [castwide/solargraph#616]
When trying to hover over a method name in Solargraph I'll get a nice hover over screen showing the method name and its full path as a link. If I click that it will try to open something like `solargraph:/document?query%3DRex%3AText%3A%3ATable.unwrap_tables%21` however this will cause the server to either crash or go into an infinite loop where it never finishes processing. I have to then restart the server again to do anything like `Go To Definition` or similar.

Would be curious if this is a known issue or if there is a plan to address this. The rest of the functionality works fine from what I can tell, but its very easy to accidentally end up triggering this behavior and then needing to manually restart the server again.

**Comment by castwide** [comment]
I haven't seen this problem. Does it happen with all documentation links, even stuff in the core/stdlib like `String`?

**Comment by gwillcox-r7** [comment]
@castwide Yes unfortunately it does. I'm attaching a video which should demonstrate the issue. This may be a VSCode plugin related issue so if that is the case I'm happy to raise the issue there.

https://streamable.com/gyih14 should showcase this issue.

**Comment by castwide** [comment]
Thanks for the update. Slow page loads are a known problem, but in my experience it's been counted in seconds, not minutes. I'll see what I can do to improve it.
